aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -498,25 +498,48 @@ std::vector<Tensor> _to_cpu(TensorList tensors) {
     return cpu_tensors;
 }
 
-Tensor to_dense_backward(const Tensor& grad, const Tensor& input_) {
+Tensor to_dense_backward(const Tensor& grad, const Tensor& input_, c10::optional<bool> masked) {
   const auto input_layout = input_.layout();
   switch (input_layout) {
     case kStrided:
+      if (masked.value_or(false)) {
+        TORCH_WARN("strided tensor to dense backward ignores masked=true");
+      }
+      // TODO: return grad as it is
       return grad.to_dense();
```

Comment:
We should probably pass `masked` to `grad.to_dense()` as well to make higher order divs correct. In fact, what the lack of `gradgradcheck`s really bothers me with sparse...

Reply:
Fixed.

Re `gradgradcheck`: it should support sparse inputs similar to gradcheck. Please create an issue if you find that it is not the case.

Reply:
It should, but I've never seen explicit `gradgracheck` calls in the tests...